third_party/terraform/tests/resource_container_cluster_test.go.erb

```diff
@@ -890,6 +890,26 @@ func TestAccContainerCluster_withSandboxConfig(t *testing.T) {
 				ImportStateVerify:       true,
 				ImportStateVerifyIgnore: []string{"min_master_version"},
 			},
+			{
+				// GKE sets automatic labels and taints on nodes. This makes
+				// sure we ignore the automatic ones and keep our own.
+				Config:   testAccContainerCluster_withSandboxConfig(clusterName),
+				PlanOnly: true,
+			},
+			{
+				// Now we'll modify the labels, which should force a change to
```

Comment:
FYI labels aren't modifiable in-place, so this test step destroys and recreates the cluster. If that's fine for this test, can you add a note here so it doesn't surprise people if they come across this?

Reply:
This is indeed intentional, and I've added a comment to note that!